web/components/stores/ClientConfigStore.tsx

```diff
@@ -38,7 +38,7 @@ let serverStatusRefreshPoll: ReturnType<typeof setInterval>;
 let hasBeenModeratorNotified = false;
 let hasWebsocketDisconnected = false;
 
-const serverConnectivityError = `Cannot connect to the Owncast service. Please check your internet connection and verify this Owncast server is running.`;
+const serverConnectivityError = `Cannot connect to the backend service. Please check your internet connection and verify this Owncast server is running.`;
```

Comment:
This was specifically called out to be the "Owncast service" to be explicit about the error. To a end user, a viewer watching the stream, to tell them "the backend is down" doesn't mean anything to them.